TestHelpers.Tests/MockDirectoryInfoTests.cs

```diff
@@ -264,5 +264,17 @@ public void MockDirectoryInfo_Constructor_ShouldThrowArgumentException_IfArgumen
             var exception = Assert.Throws<ArgumentException>(action);
             Assert.That(exception.Message, Is.StringStarting("The path is not of a legal form."));
         }
+
+        [Test]
+        [ExpectedException(typeof(PathTooLongException))]
```

Comment:
can you please use Assert.Throws<ArgumentException>() instead of [ExpectedException(typeof(PathTooLongException))]
see https://github.com/tathamoddie/System.IO.Abstractions/blob/0d3f11e508815321bbd1279a3889fe775723a39c/TestHelpers.Tests/MockFileWriteAllBytesTests.cs#L22